app/scripts/models/auth_brokers/oauth.js

```diff
@@ -17,35 +17,14 @@ define(function (require, exports, module) {
   const HaltBehavior = require('../../views/behaviors/halt');
   const OAuthErrors = require('../../lib/oauth-errors');
   const ScopedKeys = require('lib/crypto/scoped-keys');
+  const Transform = require('../../lib/transform');
   const Url = require('../../lib/url');
   const Vat = require('../../lib/vat');
 
-  /**
-   * Formats the OAuth "result.redirect" url into a {code, state} object
-   *
-   * @param {Object} result
-   * @returns {Object}
-   */
-  function _formatOAuthResult(result) {
-
-    // get code and state from redirect params
-    if (! result) {
-      return Promise.reject(OAuthErrors.toError('INVALID_RESULT'));
-    } else if (! result.redirect) {
-      return Promise.reject(OAuthErrors.toError('INVALID_RESULT_REDIRECT'));
```

Comment:
It should now be safe to comment out the [error entry in oauth-errors.js](https://github.com/mozilla/fxa-content-server/blob/fc2a86d86d66541d6037b22d5ba61f41cceede7b/app/scripts/lib/oauth-errors.js#L110). Can you leave a note about the PR # for future reference?